webapp/src/reducers/index.js

```diff
@@ -102,6 +102,33 @@ function yourRepos(state = [], action) {
     }
 }
 
+function labels(state = [], action) {
```

Comment:
Not sure if we ever read those states. If we don't, why even implement these as actions?

Reply:
@larkox You're correct, we don't need redux for label & assignee options. Here's the fix: https://github.com/mattermost/mattermost-plugin-github/pull/343/commits/a19be4e2e50ac8eb16ae778aa639320da73aa3f3